Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -28,7 +28,7 @@ Configuration
 Bundles
 ^^^^^^^
 - install ``gedmo/doctrine-extensions`` bundle in your project (check ``stof/doctrine-extensions-bundle`` for easier integration in your project) and enable the sortable feature in your config
-- install ``pixassociates/sortable-behavior-bundle`` in your project
+- install ``pixassociates/sortable-behavior-bundle`` at least version 1.1 in your project
```

Comment:
> You can use ^1.1, people will get it and copy it in their composer.json.

@greg0ire What about changing these lines into following?

Install sortable-behavior-bundle: ```composer require pixassociates/sortable-behavior-bundle ^1.1```

Same goes for gedmo/doctrine-extensions.